src/OpenTelemetry.Instrumentation.Grpc/Implementation/GrpcClientDiagnosticListener.cs

```diff
@@ -48,6 +48,7 @@ public override void OnStartActivity(Activity activity, object payload)
 
             activity.SetKind(ActivityKind.Client);
             activity.DisplayName = grpcMethod?.Trim('/');
+            activity.SetCustomProperty("GrpcHandler.Request", request);
```

Comment:
Curious - do we want this to be `OTel.GrpcHandler.Request`? For example, if someone is debugging a crash dump, they would get some hint that it was added by OpenTelemetry.

Reply:
I also think we should expose these strings as public constants in someway, part of the respective instrumentation projects, so that users dont have to remember the string property name.

Reply:
Agree to prefix "Otel."

Reply:
I added the "OTel" prefix everywhere we were calling SetCustomProperty. Moved most things to constants. But didn't tackle exposing them right now. Looked around Grpc, it doesn't have really anything that is public other than the builder extension. So we need to figure out how we want that to work. Have a constants class in each project or something? Future PR 😄

Reply:
It is unclear to me who is going to benefit from these constants. My take is that if the string literal is only going to show up in the repo once, twice or three times, literal string seems to provide better readability.